code/modules/mob/living/living_defense.dm

```diff
@@ -159,35 +159,49 @@
 		SEND_SOUND(src, sound('sound/misc/ui_toggleoffcombat.ogg', volume = 25)) //Slightly modified version of the above
 
 /mob/living/hitby(atom/movable/AM, skipcatch, hitpush = TRUE, blocked = FALSE, datum/thrownthing/throwingdatum)
```

Comment:
*later thing that should be done later* make this flags this is ridiculous. `skipcatch` is even treated like a boolean despite the default argument value being null i hate it here